__tests__/integration.test.js

```diff
@@ -81,3 +81,34 @@ describe("/api/articles/:article_id", () => {
 			});
 	});
 });
+
+describe("/api/articles/", () => {
+	test("GET:200 sends an array of article objects, with a comment count and no body property, to the client", () => {
+		return request(app)
+			.get("/api/articles")
+			.expect(200)
+			.then(({ body: { articles } }) => {
+				expect(articles.length).toBe(13);
+				articles.forEach((article) => {
+					expect(article).toMatchObject({
```

Comment:
This test is great for checking what keys _are_ there. Would be great to assert that the body key is _not_ on the response as well.